File: examples/notes/events/node/subscribeToEvents.js
const { NotesService } = require("m3o/notes");

// Specify the note to events
async function subscribeToEvents() {
  let notesService = new NotesService(process.env.MICRO_API_TOKEN);
  let rsp = await notesService.events({
    id: "63c0cdf8-2121-11ec-a881-0242e36f037a",
  });
  console.log(rsp);
}

subscribeToEvents();

File: examples/notes/events/go/subscribeToEvents.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/notes"
)

// Subscribe to notes events
func SubscribeToEvents() {
	notesService := notes.NewNotesService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := notesService.Events(&notes.EventsRequest{
		Id: "63c0cdf8-2121-11ec-a881-0242e36f037a",
	})
	fmt.Println(rsp, err)
}
